packages/core/helper-plugin/package.json

```diff
@@ -50,6 +50,7 @@
     "watch": "pack-up watch"
   },
   "dependencies": {
+    "@strapi/utils": "4.17.1",
```

Comment:
we only use the types, shouldn't this just be a `devDep`?

Reply:
sure 👍